### DISCUSSION TITLE
When is imu.heading() coming for pybricks for SPIKE Prime?

### DISCUSSION THREAD

**Discussion by lopataa** [pybricks/support#925] [General]
Hello,
I have issues with gyro on official Lego Firmware, so I wanted to move to PyBricks, but apparently `imu.heading()` isn't implemented yet. Does anyone have any estimated date when it's coming?

**Comment by laurensvalk** [comment]
A new answer can be found in https://github.com/pybricks/support/issues/989, so we can close this thread.
